src/NoFrixion.MoneyMoov/Models/Payouts/PayoutDocumentCreate.cs

```diff
@@ -0,0 +1,60 @@
+﻿// -----------------------------------------------------------------------------
+//  Filename: PayoutDocumentCreate.cs
+// 
+//  Description: Contains details of a payout document.
+// 
+//  Author(s):
+//  Axel Granillo (axel@nofrixion.com)
+// 
+//  History:
+//  28 11 2024  Axel Granillo   Created, Remote, Mexico City, Mexico.
+// 
+//  License:
+//  MIT.
+// -----------------------------------------------------------------------------
+
+using System.ComponentModel.DataAnnotations;
+using NoFrixion.MoneyMoov.Enums;
+
+namespace NoFrixion.MoneyMoov.Models;
+
+/// <summary>
+/// Document associated with a payout.
+/// </summary>
+public class PayoutDocumentCreate
+{
+    /// <summary>
+    /// Type of the document.
+    /// </summary>
+    [Required]
+    public PayoutDocumentTypesEnum DocumentType { get; set; } = PayoutDocumentTypesEnum.Other;
+    
+    /// <summary>
+    /// Used to identify the document.
+    /// </summary>
+    [Required]
+    [MaxLength(128)]
+    public string? Title { get; set; }
+    
+    /// <summary>
+    /// Additional information about the document. Optional.
+    /// </summary>
+    [MaxLength(256)]
+    public string? Description { get; set; }
+
+    /// <summary>
+    /// Currency of the document, if applicable. Optional.
+    /// </summary>
+    public CurrencyTypeEnum? Currency { get; set; }
+
+    /// <summary>
+    /// Amount of the document, if applicable. Optional.
+    /// </summary>
+    public decimal? Amount { get; set; }
```

Comment:
I'm not sure about adding these properties to `PayoutDocument` creation. If users are creating payout documents, it's because they already exist in their own system. From what I can see, a `Document` would never be created in our system first (except for Invoices in the Payruns flow, which doesn’t apply here).

With that in mind, `DocumentType` and `ExternalId` are really all they need. Additionally, I don’t think we should be storing that extra information. There could be scenarios where users update those values in one system but not the other. I believe it’s better to maintain a single source of truth (their system) and for us to only store the `ExternalId` so they can match payouts with documents in their system.

If this makes sense and we keep only `DocumentType` and `ExternalId`, then `ExternalId` should be required.

Reply:
Proceeding as discussed.

Reply:
Yep! 🤘🏻